UI/Elements/CustomUIElement.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using Terraria;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Terraria.UI;
+
+namespace BuilderEssentials.UI.Elements
+{
+    public class CustomUIElement : UIElement
```

Comment:
What is the entire point of this class? Abstract is fine and all but I don't think this is necessary.

Reply:
Adding on to that, these are clearly meant to be abstract but aren't?